__tests__/app.test.js

```diff
@@ -126,4 +126,61 @@ describe('GET /api/reviews', () => {
     });   
  })
 
- 
\ No newline at end of file
+ describe('GET /api/reviews/:review_id/comments', () => {
+    test('200: Ensures that what is passed IS an Object', () =>{ 
+        return request(app)
+        .get('/api/reviews/3/comments')
+        .expect(200)
+        .then(({body}) => {
+            console.log(body)
+            const comments = body.comments;
+            expect(comments).toBeInstanceOf(Object);
+     });
+    });
+    test('200: Returns the length of the Object passed', () =>{ 
```

Comment:
I see what you're doing here - building up from an array, to an array with a set length, to an array of objects. However, your test on line 158 implicitly checks that it's an array with the .forEach method, so we can remove these previous ones once we have confidence that it's covered by later tests.